src/store/FilingHistoryList/getters.ts

```diff
@@ -77,5 +77,23 @@ export default {
   /** Whether one document is downloading. */
   isLoadingOne (state: FilingHistoryListStateIF): boolean {
     return state.loadingOne
+  },
+
+  isCcoExpired (state: FilingHistoryListStateIF, rootGetters: any): boolean {
```

Comment:
Can you rename this getter to something like "isAuthorizedToCco" or any name which suggests what the getter is actually doing? :)
Current name indicates that if CcoExpired do something which is a bit confusing name for actual logic

Reply:
Is it possible to keep the getter name as "isAuthorizedToContinueOut" if we plan not to use the old getter? There's a getter named "isAuthorizedToContinueOut" in src/store/getters.ts. I replace it with a new getter in src/store/FilingHistoryList/getters.ts

Reply:
If the old getter is no longer used, please delete it.